.github/workflows/nix-prod-hot-fix-push.yaml

```
@@ -18,27 +18,28 @@ jobs:
   check-conditions:
     runs-on: ubuntu-latest
     outputs:
-      should-proceed: ${{ steps.evaluate.outputs.should_proceed }}      
+      should-proceed: ${{ steps.evaluate.outputs.should_proceed }}
       should-push-docker: ${{ steps.check-docker-push.outputs.should_push_docker }}
+      runner: ${{ steps.set-runner.outputs.runner }}
     steps:
       - name: Apply labels
         if: github.event_name == 'pull_request' && github.event.action != 'closed'
         uses: actions/labeler@v4
```

actionlint (1.7.12)

[error] the runner of "actions/labeler@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)